src/drivers/intel/cavs/hda-dma.c

```diff
@@ -212,7 +201,8 @@ static void hda_dma_get_dbg_vals(struct hda_chan_data *chan,
 
 #define hda_dma_ptr_trace(chan, postfix, src) \
 	do { \
-		struct hda_dbg_data *dbg_data = &(chan)->dbg_data; \
+		struct hda_chan_data *hda_chan = dma_chan_get_data(chan); \
+		struct hda_dbg_data *dbg_data = &(hda_chan)->dbg_data; \
```

Comment:
Parentheses not needed around hda_chan (it's not a macro parameter). Doesn't explain CI failures though...